### ISSUE TITLE
permission.FOREGROUND_SERVICE

### ISSUE THREAD

**Issue by SnailRabbids** [Poing-Studios/godot-admob-android#166]
### Godot version

3.5.1

### Plugin version

v2.1.0/v2.1.3

### Phone information

Android 14*

### Issue description

Hello,
Google sent an email with policy update:

"You’ll have at least 30 days from the date of this notification to update your app to comply with the following policy changes:
...blablalba...
We’re adding new guidelines to our [Device and Network Abuse policy](https://notifications.google.com/g/p/APHC3cpmVI-QRqN7pweKHVYqsQouuj9P4s7SdbxjwXSewydxXgZ1eMk8a2yOqbP6q2D-h_oAYDrb_JmL_-8BhEtbMKj8zfGgXiXxuqGQ7lIiG9tIBnM6CWb0ofxTQCAVNKDecgNFirSpjbwjUzHNSTucKiSdck0WXmKnhjN9XS-GV5-c4THCpJxc3w2U56O2p8k_sZoP_I2wk-SS-QfEqs_vFKdh2XDc1J63_7uprXBaGxkWdjM_6A):
For apps targeting Android 14 and above, a new set of policy restrictions will apply to the usage of foreground services (FGS). Apps can only declare foreground services (FGS) for appropriate use cases. Developers must also specify a valid foreground service type for each foreground service (FGS) used in your app, and declare the foreground service permission that is appropriate for that type."

As I noticed from manifest-merger-release-report, admob uses something named "androidx.work:work-runtime:2.7.0" with FOREGROUND_SERVICE permission:
```
uses-permission#android.permission.FOREGROUND_SERVICE
ADDED from [androidx.work:work-runtime:2.7.0] C:\Users\Snail\.gradle\caches\transforms-3\b85714a2827cd261ee9d0b163887676e\transformed\work-runtime-2.7.0\AndroidManifest.xml:28:5-77
```

I've tried to build my app w/o AdMob plugin and there is no "FOREGROUND_SERVICE" permission in logs.

As I understand from Google's email, all of FOREGROUND_SERVICE permissions should be reworked in something like that: 
```
  <uses-permission android:name="android.permission.FOREGROUND_SERVICE" />
  <uses-permission android:name="android.permission.FOREGROUND_SERVICE_MEDIA_PLAYBACK" />
```

What do you think about that? @gumaciel 

### Steps to reproduce

1. Export any Android project with Admob plugin
2. Open android\build\build\outputs\logs
3. See manifest-merger-release-report.txt

### Additional context

More info: https://developer.android.com/about/versions/14/changes/fgs-types-required

**Comment by gumaciel** [comment]
Ty for opening this Issue, I will investigate this soon

**Comment by gumaciel** [comment]
I will investigate this and this should be solved at v3.0.0

**Comment by gumaciel** [comment]
This is out of scope of this module, it should be done on `android\build\AndroidManifest.xml` of the project if needed
